getting-started/templates/systemlink-admin-values.yaml

```diff
@@ -1,40 +1,2 @@
-## Default values for the systemlink-admin Helm chart.
```

Comment:
We may want to update the links in @MelissaHilliard's documentation after this goes in.

Reply:
This might be more of a reason to use tags rather than moving directories? That said the tag links would go stale quickly. Maybe we just link to the releases directory?